apps/blog/src/posts/transitions-demo/transitions-demo.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Mimic Native Transitions In Your Progressive Web App
+excerpt: 
+description: 
+date: 2023-10-04
+updatedDate: 2023-10-04
+trending: true
+featured: true
+image: 
+isPost: true
+backUrl: '/'
+author:
+  name: Jaylyn Barbee
+  twitter: 
+  title: PWABuilder Engineer
+tags:
+  - PWA
+  - PWA Starter
+---
+
+Good native applications take advantage of subtle transitions from page to page that give life to the application and achieving this just became easier for web applications. 
+
+Previously, a combination of Typescript and CSS was required to replicate transitions like that of a native app, but thanks to the “CSS View Transitions” API that just became a lot easier. It’s as simple as calling .startViewTransition() and you’ve already implemented a cross fade transition from one page to another. 
+
+Choosing when and where to use these transitions in your PWA is arguably more important than using them in the first place. Implementing a consistent and accessible set of transitions across your PWA can give your app that native feel but using transitions out of place can ruin the feel of your app all together. 
+
+This blog will cover some transition tricks that to help upgrade your Progressive Web App and give it a more native feel.
+
+## Types of Transition
+
+Each type of transition has an optimal situation where it is best to use them, so before deciding which transition to use, it might be best to understand what you are trying to achieve by using the transition in the first place.  
+
+1. Are you having to load data during this transition? 
+2. How related are the pages that you are transitioning between? 
+3. Do any of the page elements need to persist when the page transitions? 
+
+The answers to these questions can help you decide what type of transition to use and when. Three common types of transition that can fit different scenarios are:
+
+1. **Top level:** Usually, a crossfade between pages with unrelated content. Seen often with navigation bars or menus. 
+2. **Container transform:** This is to be used to transition hero elements of a container from one page to another. This might look like a preview image expanding to take up more space on the hero section of the next page or title text from a store card element transitioning to the heading of a product description page. 
+3. **Forward and Backward:** This transition describes the movement from one page to another forward and backward. You see this transition a lot on settings pages where you are clicking deeper into a hierarchy. This is the classic slide in/slide out page transition. 
+
+Our main goals with transitions are to distract users from loading/page refreshes, to give life to our PWA through the movement from the transitions, and to make our PWA feel closer to that of a native application. 
```

Comment:
I suggest that you should pick one point of view and stick with it throughout your article. Before this paragraph, you have been engaging with the viewers with a 2nd person point of view - "your pwa". Here, it transitioned into a 1st person point of view - "our pwa". I always find this confusing when articles transition between these views. For strictly technical writing, I always suggest using 3rd person. Here, you can pick any, but I think it should remain to be the same.

Reply:
I will go back through and switch things to third person.